_posts/2018-08-01-mentor-spotlight-pablo-aguiar.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Mentor Spotlight: Pablo Aguiar"
+date: 2018-08-01 17:00:00
+categories: spotlights
+tags: HackIllinois2018, mentors, featured
+image: /assets/article_images/mentor-spotlight-pablo-aguiar/pablo.jpg
+author: Justin Yang
+---
+
+Welcome to our new series of mentor spotlights! In these interviews, we highlight some of the open source mentors that are central to HackIllinois.
+
+In this piece, we talked with Pablo Aguiar, who was a mentor at HackIllinois 2018. From the onset when we first approached him about mentoring at HackIllinois, Pablo's passion for open source has always shown through -- from our conversations leading up to the event to his working alongside students on open source projects at HackIllinois.
+
+### Tell us about yourself. What is your background and what do you do now?
+Hi! I’m a Computer Scientist from Brazil. In the past few years I had a lot of fun being a software engineer at Globo.com, Latin America’s largest media and publishing company. Nowadays I live in Munich, Germany, working with Electro-Mobility at The Mobility House and having fun with swarms of batteries and the Internet of Charging, reacting to the european electricity grid demands in real-time.
+
+### How did you get started in tech?
+I started programing computers at the age of 11 with lessons from my late father. He first taught me Pascal and, once I was familiar with a few algorithms and data structures, we moved to C. We had and agreement that I only could play computer games once I’ve completed my programming lessons. I’m truly grateful to him for this. Since then I never stopped programming and grew up to become a passionate Full Stack Software Engineer.
```

Comment:
- "we had and agreement" should be "we had an agreement"

- not sure how much we could grammatically change his words - if at all - because third sentence should be "We had an agreement that I only could play computer games once I had completed my programming lessons.